lora/api/metrics.go

```diff
@@ -0,0 +1,97 @@
+//
+// Copyright (c) 2018
+// Mainflux
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package api
+
+import (
+	"time"
+
+	"github.com/go-kit/kit/metrics"
+	"github.com/mainflux/mainflux/lora"
+	nats "github.com/nats-io/go-nats"
+)
+
+var _ lora.Service = (*metricsMiddleware)(nil)
+
+type metricsMiddleware struct {
+	counter metrics.Counter
+	latency metrics.Histogram
+	svc     lora.Service
+}
+
+// MetricsMiddleware instruments core service by tracking request count and
+// latency.
```

Comment:
no need to break in the two lines. Keep comment in one line as subsequent line is longer.